README.md

```diff
@@ -36,21 +36,23 @@ Open your code editor, open the folder (cloned repo) and play around with the co
 
 👤 **Oluwatoyin Olaoye**
 
-- GitHub: [@MindPrepDev](https://github.com/AbayomiOlaoye)
+- GitHub: [@AbayomiOlaoye](https://github.com/AbayomiOlaoye)
 - Twitter: [@olaoyeelijah](https://twitter.com/olaoyeelijah)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/oluwatoyinolaoye)
+- LinkedIn: [Oluwatoyin](https://linkedin.com/in/oluwatoyinolaoye)
 
+## Acknowledgement 
+ This project is premised and inspired by [Cindy Shin](https://www.behance.net/adagio07), who is the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
 
 ## Show your support 
 
 Give a ⭐️ if you like this project from a beginner like me!
 
-## Acknowledgments
+## 👍🏼Acknowledgments
 
 - Special thanks to @microverse for this opportunity
 - A worthy acknowledgment of my colleagues who presently serve as code reviewers in the institute for their constructive feedback.
 
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] This MIT link is broken. I recommend fixing this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

Reply:
Thanks for the suggestion. It's been effected.